packages/documentation/copy/en/release-notes/TypeScript 5.1.md

```diff
@@ -222,7 +222,7 @@ namespace JSX {
         // Function components
         (props: any) => Element
         // Class components
-        new (props: any) => ElementClass;
+        new (props: any) => ClassElement;
```

Comment:
I'm not sure this is right; it might be the other one that's the error. The react types contain `interface ElementClass`.

Reply:
```suggestion
        new (props: any) => ElementClass;
```